src/main/kotlin/org/koitharu/kotatsu/parsers/site/madara/vi/TruyenTranhDamMyy.kt

```diff
@@ -12,7 +12,7 @@ import org.koitharu.kotatsu.parsers.util.*
 @Broken
```

Comment:
If you had found the new link and the source works again, you must remove the @Broken mention and its import.

Reply:
okey i will edit it, sr